src/app/inventory/store/stats.ts

```diff
@@ -192,10 +209,12 @@ function buildStatsFromMods(
   for (const statHash of armorStats) {
     const hashAndValue = {
       statTypeHash: statHash,
-      value: statTracker[statHash],
+      value: _.clamp(statTracker[statHash], 0, statTracker[statHash]),
```

Comment:
`Math.max(0, statTracker[statHash])` is more concise.

Reply:
Easy